invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/MemberSearchBar.js

```diff
@@ -115,9 +115,9 @@ export class MembersSearchBar extends Component {
       this.setState({
         isFetching: false,
         suggestions: suggestions.data.hits.hits,
+        error: false
       });
     } catch (e) {
-      console.error(e);
```

Comment:
it would be nice for a developer, (or anyone) to know what was the error, because it seems not to be displayed even in the console if you remove this line ?

Reply:
that is fair, got into the habit of deleting console.logs but can be useful in this case.

Reply:
console.log != console.error so please don't delete the errors :)